src/components/delete_account/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {Trans, withTranslation} from 'react-i18next';
+import IrmaButton from '../../helpers/irma_button';
+import IrmaModal from '../../helpers/irma_modal';
+import './index.scss';
+import {connect} from 'react-redux';
+
+const mapStateToProps = state => {
+  return {
+    deleting: state.userdata.deleting,
+  };
+};
+
+class DeleteAccount extends React.Component {
+  t = this.props.t;
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      showConfirmModal: false,
+    }
+  }
+
+  onDelete() {
+    this.setState({
+      showConfirmModal: true,
+    });
+  };
+
+  onDeleteCancel() {
+    this.setState({
+      showConfirmModal: false,
+    });
+  };
+
+  onDeleteConfirm() {
+    this.props.dispatch({type: 'removeAccount'});
```

Comment:
There was no text for this in the old implementation, so that's why the TODO was there. I made a story for this on the backlog to improve this text. Doesn't seem blocking to me.